source/_integrations/eway.markdown

```
@@ -0,0 +1,171 @@
+  scan_interval: 30
+```
+
+## MQTT Topics
+The integration uses the following MQTT topic structure:
+
+- {device_id}/{device_sn}/data/post - Device data updates
+- {device_id}/{device_sn}/info/post - Device information updates
+- {device_id}/{device_sn}/data/request - Request device data
+- {device_id}/{device_sn}/info/request - Request device information
+
+### Data Format
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)